app/models/user.rb

```diff
@@ -32,6 +32,10 @@ def superadmin_or_department_manager?
     superadmin? || department_manager?
   end
 
+  def department?
+    department.first.present?
```

Comment:
because the department field gives back an array, so present? would not work on that.